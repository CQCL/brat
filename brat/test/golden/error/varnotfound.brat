f :: { Nat -> Nat }
f(x) = g(x)
