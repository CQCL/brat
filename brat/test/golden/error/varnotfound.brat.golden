Error in test/golden/error/varnotfound.brat@2:8
f(x) = g(x)
       ^

  g not found in (value) environment

